lib/items/tool-bar-button-view.js

```diff
@@ -105,37 +96,39 @@ class ToolBarButtonView extends ToolBarItem {
    * @param {ButtonOptions.callback | null} callback
    * @returns {Disposable} a disposable tooltip
    */
-  addTooltip (tooltipOptions, callback = null) {
+  addTooltip () {
+    if (!this.options.tooltip) {
+      return;
+    }
+
     let tooltip;
-    if (typeof tooltipOptions === 'string') {
+    if (typeof this.options.tooltip === 'string') {
       tooltip = {
-        title: tooltipOptions
+        title: this.options.tooltip
       };
     } else {
-      tooltip = tooltipOptions;
+      tooltip = this.options.tooltip;
     }
 
     if (!tooltip.hasOwnProperty('placement')) {
       tooltip.placement = getTooltipPlacement();
     }
 
     if (!tooltip.hasOwnProperty('keyBindingCommand') &&
-      typeof callback === 'string'
+      typeof this.options.callback === 'string'
     ) {
-      tooltip.keyBindingCommand = callback;
+      tooltip.keyBindingCommand = this.options.callback;
     }
 
-    return atom.tooltips.add(this.element, tooltip);
+    this.subscriptions = new CompositeDisposable();
+    this.subscriptions.add(atom.tooltips.add(this.element, tooltip));
```

Comment:
Please revert this. We here create a subscription inside the tooltip, while the same subscription can be used by other parts of the code. I was seeing the future here.

Reply:
Sure, already found it a bit odd that the `CompositeDisposable` was previously created inside the if-branch for the tooltip, but I refactored it like this to stay consistent with the original implementation.